examples/example_wallet_hwi_signer/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "example_wallet_hwi_signer"
+version = "0.1.0"
+edition = "2021"
+authors.workspace = true
+
+[features]
+simulator = []
+
+[dependencies]
+bdk_wallet = { path = "../../wallet", features = ["file_store"] }
+tokio = { version = "1", features = ["rt", "rt-multi-thread", "macros"] }
+bitbox-api = {version = "0.6.0", features = ["tokio", "simulator"]}
+
+async-hwi = "0.0.27"
+anyhow = "1"
+
+[patch.crates-io]
+bitbox-api = {git = "https://github.com/sdmg15/bitbox-api-rs", branch = "0.6.0-simulator-patch", features = ["tokio", "simulator"]}
```

Comment:
It looks like they already have this patched in the official release v0.8.0, we should be using that instead.

Reply:
Yes indeed I was the one that requested the fix :wink: . But it's not needed anymore. The initial way I understood what we wanted was not right.